python/paddle/fluid/tests/unittests/test_dist_ctr_with_l2_decay.py

```diff
@@ -0,0 +1,36 @@
+#   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from __future__ import print_function
+
+import os
+import unittest
+from test_dist_base import TestDistBase
+
+
+class TestDistCTR2x2(TestDistBase):
```

Comment:
Can put in `test_dist_ctr.py`

Reply:
No, if put in test_dist_ctr.py, it will run test in parallel, and the port of pserver will confilict with each other.

Reply:
Python unittest's builtin testrunner does not run tests in parallel, it's done by ctest.